### ISSUE TITLE
React-Select requires dropdowns to be searchable in order to have aria-labels

### ISSUE THREAD

**Issue by ngkelly3** [Microsoft/WebTemplateStudio#460]
In order to make accessibility requirements, dropdowns must be searchable in order to apply aria-labels.

This is a limitation of the React-Select package being used. This is a known issue and a PR is up to fix this.

https://github.com/JedWatson/react-select/pull/3090

If this is a critical issue and can't wait for this PR, then a new dropdown should be investigated or a custom one built.

**Comment by prupipho** [comment]
Having a look at this. It looks that you cannot move on the dropdown properly with the arrows, you can move using the tab and select the first or last option using Start / End (Inicio / Fin) keyboard keys.
